jerry-core/ecma/base/ecma-helpers.h

```diff
@@ -110,6 +110,11 @@
     jmem_heap_free_block ((void *) utf8_ptr, utf8_str_size); \
   }
 
+/**
+ * Convert boolean to bitfield value.
+ */
+#define ECMA_CONVERT_BOOL_TO_FIELD(x) ((x) ? 1 : 0)
```

Comment:
@robertsipka What about `ECMA_BOOL_TO_BITFIELD`? 
i) At least, for me, I can't understand what does it mean by `FIELD`.
ii) There is `ECMA_STRING_TO_UTF8_STRING` in same file. We can remove `CONVERT`.